code/_core/obj/item/clothing/head/hats/goblin_crown.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/clothing/head/hat/goblin_crown
+	name = "Goblin King's crown"
+	desc = "You monster, stealing from the little green men."
+	desc_extended = "The crown is imbued with the life essence of the late Goblin King. Greatly desired by mages."
+	icon = 'icons/obj/item/clothing/hats/goblin_crown.dmi'
+	rarity = RARITY_LEGENDARY
+
+	defense_rating = list(
+		ARCANE = AP_GREATSWORD,
+		HOLY = -AP_GREATSWORD,
+		DARK = AP_GREATSWORD
+	)
+
+	size = SIZE_2
+
+	value = 1000
+
+	var/cooldown_effect = 0
+
+/obj/item/clothing/head/hat/goblin_crown/Finalize()
+	. = ..()
+	start_thinking(src)
+
+/obj/item/clothing/head/hat/goblin_crown/think()
```

Comment:
I think it would be better if the start_thinking(src) was only called when the loincloth is worn. I'm imagining a scenario where someone has near infinite mana regen by putting 10 pairs of goblin loins in their bluespace backpack.